File: mfpeproject/refill_microservice/src/main/java/com/pack/pojo/RefillPojo.java
package com.pack.pojo;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RefillPojo {

	@JsonProperty
	private boolean statusOk;
	//private int quantity;
	private String mid;
	private long sid;
}

File: mailorder/src/model/RefillPojo.ts
export class RefillPojo{
    statusOk:boolean = false;
    mid:string='';
	sid:number = 0;

    constructor(statusOk:boolean,mid:string, sid:number)
    {
        this.statusOk = statusOk;
        this.mid = mid;
        this.sid = sid;
    }
 
}